public/css/main.css

```diff
@@ -11,6 +11,10 @@ body {
   font-family: "proxima-nova", -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen", "Ubuntu", "Cantarell", "Open Sans", "Helvetica Neue", "Arial", sans-serif;
 }
 
+body{
+   overflow-x: hidden;
+}
```

Comment:
You can just apply `overflow-x-hidden` on the body element rather than add new styles.